mu4/domain/notation/inotationcommander.h

```diff
@@ -0,0 +1,17 @@
+#ifndef INOTATIONCOMMANDER_H
+#define INOTATIONCOMMANDER_H
+
+#include "modularity/imoduleexport.h"
+
+namespace mu::domain::notation {
+class INotationCommander
+{
+public:
+    ~INotationCommander() = default;
+
+    virtual void beginCommand() = 0;
+    virtual void endCommand() = 0;
```

Comment:
Better to call it something else, for example, `INotationUndoStack` and method `prepare` and `commit`.

In MU4, the notation has no commands, only methods. Commands (action) processing is done over the notation, for example in `NotationActionController`

It's even better not to expose this interface to the outside. At the moment, management of the undostack is done only within the implementation of the notation.

For example this:
```
    adapter()->beginCommand();
    adapter()->updateStyleValue(sid, newValue);
    adapter()->endCommand();
```

Better to replace with
```
    adapter()->updateStyleValue(sid, newValue);
```
and 
```
void NotationStyle::updateStyleValue(...)
{
         startCmd();
         ...
         endCmd();
}
```

Reply:
mm, yeah, I can agree with that

Reply:
It would be great if the `INotationProperties` interface (or something) and all element property updates were hidden behind it. Maybe in the future.